Makefile

```diff
@@ -86,7 +85,7 @@ vet: ## Run go vet.
 
 .PHONY: test-unit
 test-unit: vet lint envtest ## Run unit tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./...
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test -v ./...
```

Comment:
What is the purpose of this change?

Reply:
To get more output from the unit tests, I know there aren't any currently but that could change in the future and it's nice to get more output (particularly in CI)